pkg/antctl/antctl.go

```diff
@@ -76,6 +80,19 @@ var CommandList = &commandList{
 			CommandGroup:        flat,
 			AddonTransform:      versionTransform,
 		},
+		{
+			Use:                 "agent-info",
+			Short:               "Print agent's basic information",
+			Long:                "Print agent's basic information including some connections and node subnet.",
+			HandlerFactory:      new(handlers.AgentInfo),
+			GroupVersion:        &systemGroup,
+			TransformedResponse: reflect.TypeOf(transformedAgentInfoResponse{}),
+			Agent:               true,
+			Controller:          false,
```

Comment:
nit: no need to specify negative values.

Reply:
Thanks. I have removed "Controller" and "AddTransform" fields.